src/jesse_validator_draft4.erl

```diff
@@ -69,7 +70,8 @@
                     | ?external.
 
 -type data_error_type() :: data_error()
-                         | {data_error(), binary()}.
+                         | {data_error(), binary()}
+                         | {data_error(), [jesse_error:error_reason()]}.
```

Comment:
I think `jesse_error:error_type()` needs to be updated too. (surprised it's not an alias).

`jesse_error:reason_to_jsx` might need an update as well, not sure.

Reply:
Updated the `jesse_error:error_type()`, the function `jesse_error:reason_to_jsx/1` didn't need updating.